molgenis-genomebrowser/src/main/java/org/molgenis/genomebrowser/GenomeBrowserTrack.java

```diff
@@ -0,0 +1,120 @@
+package org.molgenis.genomebrowser;
+
+import com.google.auto.value.AutoValue;
+
+import javax.annotation.Nullable;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.molgenis.data.meta.model.EntityType;
+import org.molgenis.genomebrowser.meta.GenomeBrowserAttributes;
+import org.molgenis.genomebrowser.meta.GenomeBrowserSettings;
+
+import java.util.stream.Collectors;
+
+@AutoValue
+public abstract class GenomeBrowserTrack
```

Comment:
If this POJO ever gets used as a Response Object, add `@AutoGson(autoValueClass = AutoValue_GenomeBrowserTrack.class)` below @AutoValue